src/controllers/trigger.js

```diff
@@ -27,6 +28,7 @@ const AUDITS = {
   lhs, // for all lhs variants
   404: notfound,
   'broken-backlinks': backlinks,
+  experiment,
```

Comment:
better suited term would be `experimentation` for this audit

Reply:
Accepted this change. Incorporated.